webapp/src/components/ClassAnalysisTable.tsx

```diff
@@ -71,26 +71,33 @@ const ClassAnalysisTable: React.FC<Props> = ({ jobId, pipeline }) => {
           flex: 1,
           field: "sourceClass",
           headerName: "Source Class",
+          description:
+            "The class of the samples being analyzed. For pipeline confusion, this is" +
+            " the class label.",
```

Comment:
I think the distinction for pipeline confusion is more confusing - it's always the label, no?
```suggestion
            "The class label of the samples being analyzed.",
```